scripts/commit.sh

```
@@ -9,10 +9,158 @@
+        fi
+        exit
+
+    elif [ "$choice" = "e" ] || [ "$choice" = "E" ]; then
+        if [ "$ai_mode" = "full" ] && [ -n "${msg}" ]; then
+            echo -e "${YELLOW}Edit the AI generated message:${ENDCOLOR}"
+            # Create temp file with AI message
+            commitmsg_file=$(mktemp ".commitmsg.XXXXXX")
+            echo "$ai_commit_message" > $commitmsg_file
+
+            while [ true ]; do
+                $editor $commitmsg_file
```

Shellcheck (0.10.0)

[warning] editor is referenced but not assigned.

(SC2154)